database_schemas/cassandra/benchflow.cql

```diff
@@ -23,14 +23,26 @@ CREATE TABLE database_sizes (
   trial_id text,
   dbms text,
   database_name text,
-  database_table text,
   size bigint,
   PRIMARY KEY ((trial_id, experiment_id), dbms)
 );
 
+CREATE TABLE io_data (
+  experiment_id text,
+  trial_id text,
+  container_properties_id uuid,
+  device text,
+  writes bigint,
+  reads bigint,
+  sync bigint,
+  async bigint,
+  total bigint,
+  PRIMARY KEY ((trial_id, experiment_id), container_properties_id)
+);
+
 CREATE TABLE environment_data (
   environment_data_id uuid,
-  container_properties_id uuid,
+  container_id text,
```

Comment:
@VincenzoFerme The full id of a container is very long, the likelihood of a conflict, especially considering we also have experiment id and trial id tied to them, is very unlikely. If I don't use the container id provided by docker and instead try generating my own, it will be really difficult to track considering the setup with collectors and Spark Tasks Sender messaging